.github/workflows/run-metrics.yml

```
@@ -0,0 +1,33 @@
+              print('::set-output name=insufficient_data::true')
+              print(json.dumps({'ef': None, 'drift': None}))
+              sys.exit(0)
+          ef = (w['ef_mean'].iloc[-1] - w['ef_mean'].iloc[-2]) / w['ef_mean'].iloc[-2]
+          drift = w['decoupling_mean'].iloc[-1]
+          with open(os.environ["GITHUB_OUTPUT"], "a") as fh:
+              fh.write(f"ef={ef}\n")
+              fh.write(f"drift={drift}\n")
+          print(json.dumps({'ef':ef, 'drift':drift}))
+          PY
+      - name: Fail if KPIs unmet
+        if: ${{ (steps.kpi.outputs.ef | abs) < 0.05 || steps.kpi.outputs.ef < 0 || steps.kpi.outputs.drift > 0.07 }}
```

actionlint (1.7.4)

got unexpected character ' ' while lexing || operator, expecting '|'

(expression)